src/adapters/mc/OsConfigPolicy.mof

```diff
@@ -276,7 +276,7 @@ instance of OsConfigResource as $OsConfigResource118ref
     ReportedObjectName = "auditEnsureAppropriateCiphersForSsh";
     ExpectedObjectValue = "PASS";
     DesiredObjectName = "remediateEnsureAppropriateCiphersForSsh";
-    DesiredObjectValue = "PASS";
+    DesiredObjectValue = "aes128-ctr,aes192-ctr,aes256-ctr";

```

Comment:
What happens if these values are in a different order?

Reply:
Same as for the Windows NRPs. The order of the parameter values in the policy MOF can be any. Here they are ordered like this for self-consistency.